packages/roosterjs-editor-types/lib/interface/ImageEditOptions.ts

```diff
@@ -40,10 +40,34 @@ export default interface ImageEditOptions {
      */
     imageSelector?: string;
 
+    /**
+     * Whether to use touch events due to working on mobile browsers
+     * @default false
+     */
+    isMobile?: boolean
+
+    /**
+     * customized corner resize handle style
+     * @default A predefined style
+     */
+     getCustomResizeHandelStyle?: GetResizeHandleStyle;
```

Comment:
Can you follow this pattern in TableResize plugin https://github.com/microsoft/roosterjs/blob/926ce0b1bfb26c401ec31dd33e696f48961fbb0d/packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts#L30
to allow customize more things for the resizing handle? And this can be another parameter of constructor of ImageEdit plugin, but not part of the options.

Reply:
Done. How about current implementation?